src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -427,7 +365,10 @@ int main(int argc, char **argv)
         }
     }
 
-    log_info("Starting %s\n", __FILE__);
+    if (debug)
+        Debug::Levels[MY_DEBUG_SECTION] = DBG_DATA;
```

Comment:
We can avoid adding new `debug` variable/state by moving this assignment into `case 'd'` itself. That move has a positive side effect (on future code) of enabling debugging as early as possible. I do not insist on that move.